Boundaries.h

```diff
@@ -0,0 +1,40 @@
+/* 
+ @file      Boundaries.h
+ @brief     contains the boundaries class
+ @author    Luke Eure
+ @date      January 6 2016
+*/
+
+#ifndef BOUNDARIES_H
+#define BOUNDARIES_H
+
+enum BoundaryType {
+    VACUUM,
+    REFLECTIVE,
+    BOUNDARY_NONE
+};
+
+class Boundaries {
```

Comment:
Instead of a container like this that holds multiple "boundaries"... maybe make a `Boundary` object that represents one surface/edge of the domain and then hold those objects in a simple structure.  Essentially the "object" should be _one_ boundary.

The `BoundaryType` is good though... keep that.. and just have each `Boundary` object able to be given a type.